src/Raven.Server/Documents/Handlers/Processors/Streaming/StreamingHandlerProcessorForGetDocs.cs

```diff
@@ -58,6 +59,9 @@ public StreamingHandlerProcessorForGetDocs([NotNull] DatabaseRequestHandler requ
                     },
                     initialState);
 
+                var databaseChangeVector = DocumentsStorage.GetDatabaseChangeVector(context);
+                HttpContext.Response.Headers[Constants.Headers.Etag] = "\"" + databaseChangeVector + "\"";
```

Comment:
@karmeli87 can you point me to 5.4 equivalent of this?

Reply:
the equivalent is the calculating the Etag upon loading https://github.com/ravendb/ravendb/blob/63909e7dd50eace25a2e743c5ed06a9c9166b254/src/Raven.Server/Documents/Handlers/DocumentHandler.cs#L258
The issue here is that In sharding we use the streaming EP to get documents.